src/main/java/net/dv8tion/jda/internal/requests/restaction/MessageActionImpl.java

```diff
@@ -258,6 +263,22 @@ public MessageActionImpl override(final boolean bool)
         return this;
     }
 
+    private String applyOptions(String name, AttachmentOption[] options)
```

Comment:
This could certainly be done differently if we choose to add more options in the future. This change is mostly supposed to just support that kind of change without breaking the interface further in the future.